Detectors/FIT/FDD/simulation/src/Digitizer.cxx

```diff
@@ -131,6 +131,13 @@ Double_t Digitizer::SinglePhESpectrum(Double_t* x, Double_t*)
     return 0;
   return (TMath::Poisson(y, parameters.mPMNbOfSecElec) + parameters.mPMTransparency * TMath::Poisson(y, 1.0));
 }
+//____________________________________________________________________________
+Double_t Digitizer::SignalShape(Double_t* x, Double_t*)
+{
+
+  const Double_t xx = x[0];
+  return TMath::Exp(-0.5 * TMath::Power(TMath::Log((xx + parameters.mShapeOffset) / parameters.mShapeTau) / parameters.mShapeSigma, 2));
+}
```

Comment:
a) consider declaring the input x as const `double const* x`
b) the power operation is a simple square. In this case, it is better to say
```
const double l = std::log((xx + parameters.mShapeOffset) / parameters.mShapeTau);
const double p = l*l;
return std::exp(-0.5*p);
```
otherwise the power might be accidentially be evaluated as a  sequence of logs and exps.
c) there is no added benefit of using TMath:: of std:: math functions.

Reply:
In the meantime I found that standard crystal ball function describes the test system data better than the custom function, so I excluded it completely.